routes/main.ts

```diff
@@ -3,14 +3,23 @@ import {
   BottomTabNavigationOptions,
   BottomTabScreenProps,
 } from '@react-navigation/bottom-tabs';
-import {Image} from 'react-native';
 import {RootStackParamList} from './index';
 import {ScanLayout} from '../screens/Scan/ScanLayout';
 import {HistoryScreen} from '../screens/History/HistoryScreen';
 import i18n from '../i18n';
 import {BOTTOM_TAB_ROUTES} from './routesConstants';
 import {HomeScreenLayout} from '../screens/HomeScreenLayout';
 
+let tabBarVisible = {
+  display: 'flex',
+  height: 75,
+  paddingHorizontal: 10,
+};
+
+export const changeTabBarVisible = (visible: string) => {
+  tabBarVisible.display = visible;
+};
+
```

Comment:
Can we refactor this to a CSS class and move it to theme files. Then add or remove classes accordingly.??